src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallsCollors.values().length);
+        BallsCollors color = BallsCollors.values()[index];
+        return color.toString();
     }
```

Comment:
The class name 'BallsCollors' seems to be a typo. It should follow the naming convention and be spelled correctly as 'BallColors'. Additionally, ensure that the enum is named correctly in the codebase.